src/emulator/gui/src/settings_dialog.cpp

```diff
@@ -48,13 +49,83 @@ bool change_pref_location(const std::string &input_path, const std::string &curr
     return true;
 }
 
+std::vector<std::string>::iterator modules;
+vfs::FileBuffer buffer;
+static ImGuiTextFilter search_bar;
+
 void draw_settings_dialog(HostState &host) {
     ImGui::PushStyleColor(ImGuiCol_Text, GUI_COLOR_TEXT_MENUBAR_OPTIONS);
     ImGui::Begin("Settings", &host.gui.configuration_menu.settings_dialog, ImGuiWindowFlags_AlwaysAutoResize);
     ImGuiTabBarFlags settings_tab_flags = ImGuiTabBarFlags_None;
     ImGui::BeginTabBar("SettingsTabBar", settings_tab_flags);
 
+    // Core
+    if (ImGui::BeginTabItem("Core")) {
+        ImGui::PopStyleColor();
```

Comment:
remove, push the color you need if you need another one, and pop at the end of the `if`.

Reply:
Is not possible @pent0 have help me first time usine like that is obligarory

Reply:
The push/pop is suppose for the color of the label of the tab button. I think this is valid